docs/README.rst

```diff
@@ -0,0 +1,129 @@
+.. _readme:
+
+nfs-formula
+================
+
+|img_travis| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/nfs-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/nfs-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+A SaltStack formula to install and configure nfs server and client.
+
+.. contents:: **Table of Contents**
+
+General notes
+-------------
+
+See the full `SaltStack Formulas installation and usage instructions
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+
+If you are interested in writing or contributing to formulas, please pay attention to the `Writing Formula Section
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#writing-formulas>`_.
+
+If you want to use this formula, please pay attention to the ``FORMULA`` file and/or ``git tag``,
+which contains the currently released version. This formula is versioned according to `Semantic Versioning <http://semver.org/>`_.
+
+See `Formula Versioning Section <https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#versioning>`_ for more details.
+
+If you need (non-default) configuration, please pay attention to the ``pillar.example`` file and/or `Special notes`_ section.
+
+Contributing to this repo
+-------------------------
+
+**Commit message formatting is significant!!**
+
+Please see `How to contribute <https://github.com/saltstack-formulas/.github/blob/master/CONTRIBUTING.rst>`_ for more details.
+
+Special notes
+-------------
+
+None
+
+Available states
+----------------
+
+.. contents::
+   :local:
+
+``nfs``
+^^^^^^^
+
+*Meta-state (This is a state that includes other states)*.
+
+Install nfs server components
+
+``nfs.client``
+^^^^^^^^^^^^^^
+
+Install nfs client components
+
+``nfs.mount``
+^^^^^^^^^^^^^
+
+Mount nfs shares via. pillar using the following parameters:
+* mountpoint
+* location
+* opts: default => "vers=3"
+* persist: default => True
+* mkmnt: default => True
+
+``nfs.unmount``
+^^^^^^^^^^^^^^^
+
+Unmount nfs shares via. pillar using the following parameters:
+* mountpoint
```

Comment:
Need an empty line before the bullet list because it's isn't being rendered properly:

* https://github.com/saltstack-formulas/nfs-formula/blob/13cd90155eaa879faa97a6cfb341ebaf233f4c31/docs/README.rst#nfsunmount